### PR TITLE
Added Mapbox Studio

### PR DESCRIPTION
**Pull request by ducky427**
Its a pretty complex electron app. It looks like its very complete.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Some good apps written with Electron.
 - [imagemin-app](https://github.com/imagemin/imagemin-app) - Minify images.
 - [LevelUI](https://github.com/hij1nx/levelui) - LevelDB management.
 - [Douban-FM-Express](https://github.com/cyrilis/Douban-FM-Express) - DoubanFM client.
-
+- [Mapbox Studio](https://github.com/mapbox/mapbox-studio) - Mapbox Studio
```
Comment (sindresorhus) [inline readme.md]: missing a end dot

**Comment by ducky427** [comment]
Added the fullstop.

File: readme.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Some good apps written with Electron.
 - [imagemin-app](https://github.com/imagemin/imagemin-app) - Minify images.
 - [LevelUI](https://github.com/hij1nx/levelui) - LevelDB management.
 - [Douban-FM-Express](https://github.com/cyrilis/Douban-FM-Express) - DoubanFM client.
-
+- [Mapbox Studio](https://github.com/mapbox/mapbox-studio) - Mapbox Studio.
```
Comment (sindresorhus) [inline readme.md]: Description is just the name, this would be better:

> Vector tile driven map design.

**Comment by sindresorhus** [comment]
fixed in https://github.com/sindresorhus/awesome-electron/commit/6be8b6a7ac9dc5b496443188495f316ed1e0448e

**Comment by ducky427** [comment]
Thank you!

**Pull request closed without merging**